zeppelin-web/src/app/jobmanager/jobs/job.html

```diff
@@ -0,0 +1,107 @@
+      class="job-types"
+      ng-switch="notebookJob.notebookType">
+      <i ng-switch-when="normal" class="icon-doc"></i>
+      <i ng-switch-when="cron" class="icon-clock"></i>
+      <i ng-switch-default class="icon-question"></i>
+    </span>
+    &nbsp;
+    <a style="text-decoration: none !important;" ng-href="#/notebook/{{notebookJob.notebookId}}">
+      <span>
+       {{notebookJob.notebookName}}
+      </span>
+      <span>
+        &nbsp;-&nbsp;
+      </span>
+      <span>
+        <span ng-if="notebookJob.interpreter === undefined" style="color: orange">
+          Interpreter is not set
+        </span>
+        <span ng-if="notebookJob.interpreter !== undefined" style="color: gray">
+          {{notebookJob.interpreter}}
+        </span>
+      </span>
+    </a>
+    <div ng-include src="'app/jobmanager/jobs/job-progressBar.html'"></div>
+  </div>
+
+  <div>
+    <span
+      ng-repeat="paragraphJob in notebookJob.paragraphs"
+      ng-switch="paragraphJob.status">
+      <a ng-switch-when="READY"
+         style="text-decoration: none !important;"
+         ng-href="#/notebook/{{notebookJob.notebookId}}/paragraph/{{paragraphJob.id}}">
+        <i style="color: green" class="fa fa-circle-o"
+           tooltip-placement="top-left"
+           tooltip="{{paragraphJob.name}} is READY">
+        </i>
+      </a>
+      <a ng-switch-when="FINISHED"
+         style="text-decoration: none !important;"
+         ng-href="#/notebook/{{notebookJob.notebookId}}/paragraph/{{paragraphJob.id}}">
+        <i style="color: green" class="fa fa-circle"
+           tooltip-placement="top-left"
+           tooltip="{{paragraphJob.name}} is FINISHED">
+        </i>
+      </a>
+      <a ng-switch-when="ABORT"
+         style="text-decoration: none !important;"
+         ng-href="#/notebook/{{notebookJob.notebookId}}/paragraph/{{paragraphJob.id}}">
+        <i style="color: orange" class="fa fa-circle"
+           tooltip-placement="top-left"
+           tooltip="{{paragraphJob.name}} is ABORT">
+        </i>
+      </a>
+      <a ng-switch-when="ERROR"
+         style="text-decoration: none !important;"
+         ng-href="#/notebook/{{notebookJob.notebookId}}/paragraph/{{paragraphJob.id}}">
+        <i style="color: red" class="fa fa-circle"
+           tooltip-placement="top-left"
+           tooltip="{{paragraphJob.name}} is ERROR">
+        </i>
+      </a>
+      <a ng-switch-when="PENDING"
+         style="text-decoration: none !important;"
+         ng-href="#/notebook/{{notebookJob.notebookId}}/paragraph/{{paragraphJob.id}}">
+        <i style="color: gray" class="fa fa-circle"
+           tooltip-placement="top-left"
+           tooltip="{{paragraphJob.name}} is PENDING">
+        </i>
+      </a>
+      <a ng-switch-when="RUNNING"
+         style="text-decoration: none !important;"
+         ng-href="#/notebook/{{notebookJob.notebookId}}/paragraph/{{paragraphJob.id}}">
+        <i style="color: blue" class="fa fa-spinner spinAnimation"
```

Comment:
In Chrome, it does not animate 'fa-spinner'. How about `fa fa-spinner fa-spin` to animate in chrome?